client/android/project/build.gradle

```diff
@@ -0,0 +1,116 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+buildscript {
+    repositories {
+        jcenter()
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:2.1.2'
+        classpath 'com.github.dcendents:android-maven-gradle-plugin:1.3'
+        classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.7'
+        // NOTE: Do not place your application dependencies here; they belong
+        // in the individual module build.gradle files
+    }
+}
+
+allprojects {
+    repositories {
+        jcenter()
+    }
+    apply plugin: 'com.android.library'
+    apply plugin: 'com.jfrog.bintray'
+    apply plugin: 'com.github.dcendents.android-maven'
+}
+
+android {
+    compileSdkVersion 23
+    buildToolsVersion "23.0.3"
+
+    defaultConfig {
+        minSdkVersion 7
+        targetSdkVersion 23
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+
+    dependencies {
+       compile fileTree(dir: 'libs', include: ['*.jar'])
+       testCompile 'junit:junit:4.12'
+    }
+}
+
+//Run with JDK 8
+def release_version = '1.0.0'
+def release_desc = 'Catbag/Switchboard forked from KeepSafe/Switchboard first release'
+def release_group = 'br.com.catbag.switchboard'
+
+group = release_group
+version = release_version
+Properties properties = new Properties()
+properties.load(project.rootProject.file('local.properties').newDataInputStream())
+bintray {
+    //Put this two on local.properties
+    //bintray.user=<your user>
+    //bintray.apikey=<your api key>
+    user = properties.getProperty("bintray.user")
+    key = properties.getProperty("bintray.apikey")
```

Comment:
Its required for all users that want download this module, have an account in bintray?